build-s2i-python-kopf/version.json

```diff
@@ -1 +1 @@
-{"version":"v0.28.2"}
+{"version":"v0.27.0"}
```

Comment:
Version decrement?

Reply:
Yeah... this is a weird case. The version wasn't supposed to be 0.28 to begin with. The plan was to match the version to the python kopf version, which is now at 0.27. I'm not sure how I ended up creating the image version at 0.28. I'm torn about whether I should continue at 0.28 until kopf catches up, or go ahead and just match the version immediately like this. What do you think?

Reply:
I suppose waiting for kopf to catch up is probably the more obvious choice and less likely to create mystery...